### PR TITLE
#126 chore: added workspacresource linking to repository and description

### PR DESCRIPTION
**Pull request by zsinnema**

### CODE REVIEW

**Comment by zsinnema** [comment]
@filippomc please don't remove the feature/126 branch I'm still working in it to implement the other stuff (repository volume creation etc)

File: applications/workspaces/api/openapi.yaml [hunk 1]
```diff
@@ -98,8 +98,8 @@ paths:
       tags:
         - rest                  
       summary: Used to save a Workspace to the repository. The user_id (keycloak user id) will be automatically filled with the current user
-      security:
-        - bearerAuth: []
+      # security:
```
Comment (filippomc) [inline applications/workspaces/api/openapi.yaml:101]: Was this commented for testing or on purpose?

File: applications/workspaces/api/openapi.yaml [hunk 2]
```diff
@@ -260,17 +260,37 @@ paths:
...
     post:
       tags:
-        - rest                  
+        - rest
+      operationId: post
+      x-openapi-router-controller: workspaces.controllers.workspace_resource_controller
       summary: Used to save a WorkspaceResource to the repository.
-      security:
-        - bearerAuth: []
+      # security:
```
Comment (filippomc) [inline applications/workspaces/api/openapi.yaml:267]: Comment for testing?

File: applications/workspaces/api/openapi.yaml (hunk 1 above)
Comment (zsinnema) [reply applications/workspaces/api/openapi.yaml:101]: yes this is for testing

File: applications/workspaces/api/openapi.yaml (hunk 2 above)
Comment (zsinnema) [reply applications/workspaces/api/openapi.yaml:267]: testing

**Review by filippomc — CHANGES_REQUESTED** [review]
Found a few bugs:
POST osbrepository gives error 500 if auto_sync is false
GET osbrepository contentTypesList is not set

**Comment by zsinnema** [comment]
> Found a few bugs:
> 
> * POST osbrepository gives error 500 if auto_sync is false
> * GET  ​osbrepository contentTypesList is not set
> * GET  ​osbrepository user is not set (Need to show the user name in the list)

fixed

**Comment by zsinnema** [comment]
> Also would make life easier in the frontend having a generic type representing paginated content

@filippomc do you want to do this right now? Or is this something for later?

**Pull request merged**